dali/pipeline/operator/op_schema.h

```diff
@@ -90,6 +92,65 @@ class DLL_PUBLIC OpSchema {
     return *this;
   }
 
+  /**
+   * @brief Sets the docstring for input.
+   *
+   * Set the documentation for intput at given `index`.
+   *
+   * If the operator specifies some range of allowed inputs with NumInput(int min, int max)
+   * only the first `min` inputs are considered mandatory, the rest are optional
+   *
+   * Will generate entry in `Args` section using numpydoc style:
+   * `name`: type_doc
+   *     doc
+   */
+  DLL_PUBLIC inline OpSchema &InputDocStr(int index, const string &name, const string &type_doc,
```

Comment:
Rename to InputDocs or InputDox - the actual InputDocStr is built later, not set here explicitly.

Reply:
Not fond of this change, but done.